src/linear_algebra/projective_space/basic.lean

```diff
@@ -58,13 +62,23 @@ def mk (v : V) (hv : v ≠ 0) : ℙ K V := quotient.mk' ⟨v,hv⟩
 /-- A variant of `projectivization.mk` in terms of a subtype. `mk` is preferred. -/
 def mk' (v : { v : V // v ≠ 0 }) : ℙ K V := quotient.mk' v
 
-@[simp] lemma mk'_eq_mk (v : { v : V // v ≠ 0}) :
+@[simp]
+lemma mk'_eq_mk (v : { v : V // v ≠ 0}) :
   mk' K v = mk K v v.2 :=
 by { dsimp [mk, mk'], congr' 1, simp }
 
+/- Since `mk` is the preferred representation, we can `simp` others to it. -/
+
+@[simp]
+lemma quotient_mk_eq_mk (z : V) (h : z ≠ 0) :
+  @quotient.mk _ (projectivization_setoid _ _) ⟨z, h⟩ = mk K z h := rfl
```

Comment:
This probably makes more sense with the `mk'` variant; things have gone quite wrong if the unprimed variant appears.

Reply:
I agree, the thing is at some point when using the API (I don't exactly remember the circumstances, but I think it had to do with using topological lemmas applied to the quotient topology) some bracket notation `⟦⟨x,hx⟩⟧` came up and this is the normalization I needed at the time. The whole point is to normalize everyone to the `projectivization.mk` version.

Adding the `mk'` version would make sense, but it breaks a later proof that uses `simp [mk]` so it may require rewriting a few things. Admittedly having to `simp [mk]` is the sign of missing API which the `mk'` version is intended to help provide. I will give it a try.

Reply:
> I think it had to do with using topological lemmas applied to the quotient topology

Perhaps there are some primed lemmas missing there

Reply:
Added, and I rewrote 2 proofs further down.